<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window1</class>
 <widget class="QWidget" name="Window1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CanSatApp</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="dbView"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Внести данные в таблицу</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="showButton">
         <property name="text">
          <string>Показать таблицу</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="divButton">
         <property name="text">
          <string>Умножить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="chooseButton">
         <property name="text">
          <string>Выбрать таблицу</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Умножить столбец под номером:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="columnBox">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>На 10^n, где n =</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nBox">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
